src/components/Header/Header.jsx

```diff
@@ -0,0 +1,255 @@
+import React, { useState } from "react"
+import { useLocation } from 'react-router-dom'
+import { NavLink } from "react-router-dom"
+import CyrillicToTranslit from 'cyrillic-to-translit-js'
+import ListenerWidth from "./ListenerWidth"
+import {
+    ContainerHeder, LogoHeader, LeftBlock, PointsBalance, NumberBalance,
+    MenuHeaderBlock, MenuContact, MenuSection, MenuInitiaLetterCircle, MenuInitiaLetter, MenuName,
+    MobileHamburgerContainer, MobileHamburgerLine, MobileHamburgerbackground, MobileHamburgerLoginBox,
+    MobileHamburgerCrossBox, MobileHamburgerCross, MobileHamburgerCross1,
+    MobileHamburgerSectionMenuUp, MobileHamburgerSectionMenu
+} from "./Header.styled"
+
+import { ReactComponent as Logo } from "../../assets/image/HeaderKidsLogo.svg"
+import { ReactComponent as ExitIcon } from "../../assets/image/HeaderNavExitIcon.svg"
+import { ReactComponent as ExitIconMobile } from "../../assets/image/HeaderNavExitIconMobile.svg"
+
+const UserBalance = 123
+const UserName = 'UserName'
+const logOut = () => localStorage.clear()
+
+const cyrillicToTranslit = new CyrillicToTranslit()
+const firstLetter = UserName && (UserName.charAt(0))
+const nameLetter = cyrillicToTranslit.transform(firstLetter).toUpperCase()
```

Comment:
It is easier to convert by CSS property text-transform

Reply:
Thanks for the advice) I will study the possibility of such an implementation.